app/views/experiments/_assigned_users.html.erb

```diff
@@ -1,4 +1,4 @@
-<% users = my_module.designated_users.order(:full_name) %>
+<% users = my_module.assigned_users.order(:full_name) %>
```

Comment:
This is not correct. Designated users is a different thing from assigned users.
What needs to be done here is to also clear the user from designated users along with the assignments.